.circleci/config.yml

```diff
@@ -447,22 +447,26 @@ jobs:
               || echo "Skipping push since it looks like there were no changes"
           # yamllint enable rule:line-length
   deploy-to-private-gcr:
-    executor: gcp-gcr/default
+    executor: docker/docker
```

Comment:
The default gcp-gcr executor looks to be `machine` with defaults: https://circleci.com/developer/orbs/orb/circleci/gcp-gcr#executors-default

To keep things as similar as possible, I think this could be:

```
executors:
  my-executor:
    machine:
      image: ubuntu-2004:202111-02
```

Perhaps that would avoid having to make some of the changes below?

If this all gets too complicated, I wouldn't be opposed to reverting back to adding jdk to the Dockerfile.